__init__.py

```diff
@@ -162,6 +162,12 @@ def _handle_client_exception(self, callback, *args, **kwargs):
         return False
 
     # Intent handlers
+    @intent_handler('show.camera.image.intent')
+    def handle_show_camera_image_intent(self, message):
+        """Handle show camera image intent."""
```

Comment:
Hi @pfefferle, would You add also type annotations as I did in my last PR so we keep same style?

```
def handle_show_camera_image_intent(self, message: Message) -> None:
       """Handle show camera image intent."""
```

Reply:
sure!